checks/evaluation/webhooks_test.go

```diff
@@ -32,104 +32,105 @@ func TestWebhooks(t *testing.T) {
-				r:    &checker.WebhooksData{},
+			name: "1 webhook with no secret",
+			findings: []finding.Finding{
+				{
+					Probe:   "hasWebhooks",
+					Outcome: finding.OutcomePositive,
+				}, {
+					Probe:   "webhooksWithoutTokenAuth",
+					Outcome: finding.OutcomeNegative,
+				},
 			},
-			want: checker.CheckResult{
-				Score: checker.MaxResultScore,
+			result: scut.TestReturn{
+				Score: 0,
 			},
 		},
 		{
 			name: "1 webhook with secret",
-			args: args{
-				name: "1 webhook with secret",
-				dl:   &scut.TestDetailLogger{},
-				r: &checker.WebhooksData{
-					Webhooks: []clients.Webhook{
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/1234",
-							ID:             1234,
-							UsesAuthSecret: true,
-						},
-					},
+			findings: []finding.Finding{
+				{
+					Probe:   "hasWebhooks",
+					Outcome: finding.OutcomePositive,
+				}, {
+					Probe:   "webhooksWithoutTokenAuth",
+					Outcome: finding.OutcomePositive,
 				},
 			},
-			want: checker.CheckResult{
+			result: scut.TestReturn{
 				Score: 10,
 			},
 		},
 		{
-			name: "1 webhook with no secret",
-			args: args{
-				name: "1 webhook with no secret",
-				dl:   &scut.TestDetailLogger{},
-				r: &checker.WebhooksData{
-					Webhooks: []clients.Webhook{
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/1234",
-							ID:             1234,
-							UsesAuthSecret: false,
-						},
-					},
+			name: "2 webhooks one of which has secret",
+			findings: []finding.Finding{
+				{
+					Probe:   "hasWebhooks",
+					Outcome: finding.OutcomePositive,
+				},
+				{
+					Probe:   "hasWebhooks",
+					Outcome: finding.OutcomePositive,
+				},
+				{
+					Probe:   "webhooksWithoutTokenAuth",
+					Outcome: finding.OutcomeNegative,
 				},
 			},
-			want: checker.CheckResult{
-				Score: 0,
+			result: scut.TestReturn{
+				Score: 5,
 			},
 		},
 		{
-			name: "many webhooks with no secret and with secret",
-			args: args{
-				name: "many webhooks with no secret and with secret",
-				dl:   &scut.TestDetailLogger{},
-				r: &checker.WebhooksData{
-					Webhooks: []clients.Webhook{
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/1234",
-							ID:             1234,
-							UsesAuthSecret: false,
-						},
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/1111",
-							ID:             1111,
-							UsesAuthSecret: true,
-						},
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/4444",
-							ID:             4444,
-							UsesAuthSecret: true,
-						},
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/3333",
-							ID:             3333,
-							UsesAuthSecret: false,
-						},
-						{
-							Path:           "https://github.com/owner/repo/settings/hooks/2222",
-							ID:             2222,
-							UsesAuthSecret: false,
-						},
-					},
+			name: "Five webhooks three of which have secrets",
```

Comment:
add a test with more than 10 failing webhooks, to verify the score is 0 and not negative